src/addrman.cpp

```diff
@@ -15,37 +15,27 @@
 #include <unordered_map>
 #include <unordered_set>
 
-//! over how many buckets entries with tried addresses from a single group (/16 for IPv4) are spread
+/** Over how many buckets entries with tried addresses from a single group (/16 for IPv4) are spread */
 static constexpr uint32_t ADDRMAN_TRIED_BUCKETS_PER_GROUP{8};
-
-//! over how many buckets entries with new addresses originating from a single group are spread
+/** Over how many buckets entries with new addresses originating from a single group are spread */
 static constexpr uint32_t ADDRMAN_NEW_BUCKETS_PER_SOURCE_GROUP{64};
-
-//! in how many buckets for entries with new addresses a single address may occur
+/** Maximum number of times an address can be added to the new table */
```

Comment:
nit: I think the old comment was more precise in referring to state rather than process ("added") - an address could be added more often over time, in the not untypical case where earlier instances were replaced due to new table collisions.

Reply:
ah, I see what you're saying. maybe this would be better?

```suggestion
/** Maximum number of times an address can occur in the new table */
```

I'm not planning to address right now, but will keep it in mind for future changes :)